### PR TITLE
[stable/postgres] add priority class name option to postgres chart

### PR DESCRIPTION
**Pull request by jesstracy**
Signed-off-by: Jessica Tracy <j.tracy916@gmail.com>

maintainers:
@desaintmartin 
@Bitnami
#### Is this a new chart
No

#### What this PR does / why we need it:

Adds the option to include a priority class name (https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/) to each pod in the master statefulset and/or the slave statefulset.

#### Which issue this PR fixes
N/A

#### Special notes for your reviewer:
Thank you! Any feedback welcome!

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for the contribution, before continue, you need to [sign your existing commits](https://github.com/apps/dco), you can find custom instructions in the “Details” link from the below DCO check, also you should increment the chart version in the `Chart.yml` according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

In this case, as you're adding new functionality, can you bump the chart version in a minor?

File: stable/postgresql/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 8.0.0
+version: 8.0.1
```
Comment (carrodher) [inline stable/postgresql/Chart.yaml]:
```suggestion
version: 8.1.0
```

**Review by carrodher — CHANGES_REQUESTED** [review]
Thanks for your contribution! 👏 
Before continue, you need to [sign your existing commits](https://github.com/apps/dco), you can find custom instructions in the “Details” link from the below DCO check, also you should increment the chart version in the `Chart.yml` according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

In this case, as it is a new functionality, can you bump the chart version in a minor?

In the same way, this chart also contains a [`values-production.yaml`](https://github.com/helm/charts/blob/master/stable/postgresql/values-production.yaml) file, please, can you implement the same changes in this file?

**Comment by jesstracy** [comment]
Thanks @carrodher!! I made the changes, let me know if there's anything else I missed :)

**Pull request merged**